section1/reflection.md

```diff
@@ -1,19 +1,33 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+    * I tend to wait too long to ask questions about certain issues that I am stuck on. Usually this is because of me believing that I have a simple fix and do not want to sound dumb, or un educated on the subject I am working on. I have recently tried to ask more questions, and I believe that once I come out of my shell I begin to ask alot of questions in order to truly understand the topics that are covered.
```

Comment:
Yes I can understand that. 

I find "rubber ducking" to be very helpful -talking to yourself or a rubber duck or other inanimate object - almost rehearsing the question you are going to ask/explain the situation - many times you'll find you can answer it yourself just by slowing down to present all the info.